transcode/manifest.go

```diff
@@ -64,7 +64,7 @@ func GetSourceSegmentURLs(sourceManifestURL string, manifest m3u8.MediaPlaylist)
 			urls,
 			SourceSegment{
 				URL:            u,
-				DurationMillis: int64(segment.Duration),
+				DurationMillis: int64(segment.Duration * 1000),
```

Comment:
This is where the issue was - the duration in the source manifest is in Seconds

Reply:
```suggestion
				DurationMillis: int64(segment.Duration * time.Second),
```

WDYT?

Reply:
Much nicer!